quickwit/Cargo.toml

```diff
@@ -238,3 +238,11 @@ encoding_rs = "=0.8.29"
 
 [patch.crates-io]
 sasl2-sys = { git = "https://github.com/quickwit-oss/rust-sasl/", rev = "daca921" }
+
+#tracing = { git = "https://github.com/trinity-1686a/tracing.git", branch = "tracing-subscriber-0.3.16-poll-duration" }
```

Comment:
do we want to keep those?

Reply:
yes. This is what provides the changes on traces (visible on Jaeger). I should probably move the fork to `quickwit-oss/` though

Reply:
thanks for adding the comment